source/_integrations/simplisafe.markdown

```diff
@@ -153,6 +153,29 @@ triggers:
         last_event_type: doorbell_detected
 ```
 
+### Using Secret Alerts for Sensor Changes
+
+For cases where the default poll time of 30 seconds is too long for creating automations, you can use Secret Alerts to get push notifications of a sensor being triggered.
```

Comment:
```suggestion
For cases where the default {% term polling %} interval of 30 seconds is too long for automations, you can use secret alerts to get push notifications of a sensor being triggered.
```
This adds a reference to a [glossary](https://www.home-assistant.io/docs/glossary/) term. It will show up as a tooltip.